mojio_sdk/fuel.py

```diff
@@ -0,0 +1,12 @@
+class Fuel:
+
+    def __init__(self, json_data):
+        self.fuel_level = 0.0
```

Comment:
Would something like this work? (I'm by no means a Python expert, I just googled that).

`self.fuel_level = 0.0 if json_data.get('FuelLevel') is None else json_data.get('FuelLevel').get('Value', 0.0)`

I prefer one-liners.

Reply:
This is the last thing to address, I'll get to it this weekend.   Regards.